<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profileEditPage</class>
 <widget class="QWidget" name="profileEditPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>293</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="listLayout">
       <item>
        <widget class="KListWidget" name="profilesList">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>334</width>
          <height>283</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionNewProfile">
   <property name="text">
    <string>New Profile</string>
   </property>
  </action>
  <action name="actionDeleteProfile">
   <property name="text">
    <string>Delete Profile</string>
   </property>
  </action>
  <action name="actionImportProfiles">
   <property name="text">
    <string>Import Profiles</string>
   </property>
  </action>
  <action name="actionExportProfiles">
   <property name="text">
    <string>Export Profiles</string>
   </property>
  </action>
  <action name="actionRestoreDefaultProfiles">
   <property name="text">
    <string>Restore Default Profiles</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
